main.rb

```diff
@@ -0,0 +1,39 @@
+module Main
```

Comment:
- [ ] Kindly change this module name to `Utils` as that is the name you've been using in your `app.rb` file.

Reply:
resolved 👍